editor/src/clj/editor/workspace.clj

```diff
@@ -50,16 +51,12 @@ ordinary paths."
 
 (set! *warn-on-reflection* true)
 
-;; Class loader used when loading editor extensions from libraries.
-;; It's important to use the same class loader, so the type signatures match.
-(def ^DynamicClassLoader class-loader (DynamicClassLoader. (.getContextClassLoader (Thread/currentThread))))
-
-(defn load-class! [class-name]
-  (Class/forName class-name true class-loader))
-
 (def build-dir "/build/default/")
 (def plugins-dir "/build/plugins/")
 
+;; SDK api
+(def load-class! java/load-class!)
+
```

Comment:
Turns out I can't just remove `load-class!` from the workspace ns because [other libraries use it](https://github.com/search?q=workspace%2Fload-class%21&type=code). They don't use `class-loader` though!